pkg/k8shandler/status.go

```diff
@@ -291,7 +291,7 @@ func (clusterRequest *ClusterLoggingRequest) getPodConditions(component string)
 						Status:             v1.ConditionTrue,
 						Reason:             containerStatus.State.Terminated.Reason,
 						Message:            containerStatus.State.Terminated.Message,
-						LastTransitionTime: metav1.Now(),
+						LastTransitionTime: containerStatus.State.Terminated.FinishedAt,
```

Comment:
what happens when the state is other then "terminated"? Isn't it possible we are checking this condition when the pod is still running

Reply:
@jcantrill this block is only when the container status state is terminated. there are other blocks for other conditions